<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite1" duration-ms="34295" started-at="2018-08-05T02:55:23Z" finished-at="2018-08-05T02:55:57Z">
    <groups>
    </groups>
    <test name="Webdriver" duration-ms="34295" started-at="2018-08-05T02:55:23Z" finished-at="2018-08-05T02:55:57Z">
      <class name="TestNG.NewTest3">
        <test-method status="PASS" signature="verifytitle1()[pri:0, instance:TestNG.NewTest3@29ee9faa]" name="verifytitle1" duration-ms="12828" started-at="2018-08-05T02:55:45Z" finished-at="2018-08-05T02:55:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifytitle1 -->
      </class> <!-- TestNG.NewTest3 -->
      <class name="TestNG.NewTest2">
        <test-method status="PASS" signature="launch()[pri:0, instance:TestNG.NewTest2@6767c1fc]" name="launch" is-config="true" duration-ms="19103" started-at="2018-08-05T02:55:23Z" finished-at="2018-08-05T02:55:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launch -->
        <test-method status="PASS" signature="verfiytitle()[pri:0, instance:TestNG.NewTest2@6767c1fc]" name="verfiytitle" duration-ms="12" started-at="2018-08-05T02:55:42Z" finished-at="2018-08-05T02:55:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verfiytitle -->
        <test-method status="FAIL" signature="verfiyurl()[pri:0, instance:TestNG.NewTest2@6767c1fc]" name="verfiyurl" duration-ms="12" started-at="2018-08-05T02:55:42Z" finished-at="2018-08-05T02:55:42Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [edureka] but found [https://www.edureka.co/]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [edureka] but found [https://www.edureka.co/]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at TestNG.NewTest2.verfiyurl(NewTest2.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verfiyurl -->
        <test-method status="PASS" signature="close()[pri:0, instance:TestNG.NewTest2@6767c1fc]" name="close" is-config="true" duration-ms="2317" started-at="2018-08-05T02:55:42Z" finished-at="2018-08-05T02:55:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- TestNG.NewTest2 -->
    </test> <!-- Webdriver -->
  </suite> <!-- Suite1 -->
</testng-results>
